telethon/_client/telegramclient.py

```diff
@@ -73,6 +73,21 @@ class TelegramClient:
             You only need to use this if you have multiple network cards and
             want to use a specific one.
 
+        default_dc_id (`int`, optional):
+            The DC ID to connect to, if not already provided by the session.
+
+        default_ipv4_ip (`str`, optional):
+            The default IPv4 address to connect to, if not already provided by
+            the session.
+
+        default_ipv6_ip (`str`, optional):
+            The default IPv6 address to connect to, if not already provided by
+            the session.
+
+        default_port (`int`, optional):
+            The default port to connect to, if not already provided by
+            the session. This is used whether the connection is IPv4 or IPv6.
```

Comment:
I'm not sure how common this would be. What is the use-case? Should this really be per-client or affect the entire library (with a different free-standing method to override this; much like how message's parse mode can be globally set).

Reply:
Testmode support. Removing this would be a major regression as there are people who bridge non-testmode to testmode using a single application.